exercises/isogram/Isogram.example.elm

```diff
@@ -5,16 +5,16 @@ import List
 import Char
 
 
-isLetter : Char -> Bool
-isLetter c =
-    Char.isHexDigit c && (not <| Char.isDigit c)
+isAlpha : Char -> Bool
+isAlpha char =
```

Comment:
same implementation as the future elm-core Char module

https://github.com/elm-lang/core/blob/master/src/Char.elm#L86